ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_ingestauth_key.schema.yml

```diff
@@ -10,5 +10,7 @@ keys:
     type: str
     description: |-
       On-premise CVP ingest auth key. If set, TerminAttr will be configured with key-based authentication for on-premise CVP.
-      If not set, TerminAttr will be configured with certificate based authentication using token-secure onboarding.
+      If not set, TerminAttr will be configured with certificate based authentication:
+      - On-premise using token onboarding. Default token path is '/tmp/token'.
+      - CVaaS using token-secure onboarding. Default token path is '/tmp/cv-onboarding-token'.
```

Comment:
we should take th eopportunity of this PR to fix the description for `cvp_instance_ips:`


```
description: 'You can either provide a list of IPs to target on-premise CloudVision
      cluster or
      use DNS name for your CloudVision as a Service instance. If you have both on-prem
      and
      CVaaS defined, only on-prem is going to be configured.
      '
```

Our code actually will use CVaaS only if `arista.io` is insisde the domain name. Otherwise we use on-prem.

Also I am not quite sure that if we have both on-prem and CVaaS defined, only on-prem is configured looking at the code. Looking at the for loop it looks to me that depending on the order of the ip addresses mixing arista.io and not, the outcome can be different

Reply:
Ah sorry I didn't see you were pointing to an unrelated schema. That will be touched on in the schema PR for management. Let's not bring it in here.